client/js/presenter.js

```diff
@@ -117,9 +136,35 @@ this.subscribeToEvents= function (){
   .on('asq:question_type', function(evt){
     console.log(evt);
     debug.log(evt)
+  })
+  .on('setting:update-live-presentation-settings', updateLivePresentationSettings);
+}
+
+function updateExerciseSettings(exercise, settings) {
+  Object.keys(settings).forEach(function(key, index){
+    exercise.setAttribute(key.toLowerCase(), settings[key]);
+    exercise[key] = settings[key];
   });
 }
 
+function updateLivePresentationSettings(evt) {
```

Comment:
This function selects all asq-exercise elements from the DOM and then sets their attribute. This is the tight coupling we wanted to avoid using the eventemitter object. asq-exercise elements should add a listener for the settings event and update themselves.